### PR TITLE
Adding workshop tutorial link to the README

### PR DESCRIPTION
**Pull request by fredgalvao**
## Motivation and Context
Hoping someone will eventually read it, or the docs, :wink:.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,8 @@ This plugin offers support to receive and handle native push notifications with
 - [Push notification payload details](docs/PAYLOAD.md)
 - [Contributing](.github/CONTRIBUTING.md)
 - [License (MIT)](MIT-LICENSE)
+
+
+# Do you like tutorial? You get tutorial!
+
+ - [PhoneGap Day EU/US Push Workshop 2016](http://macdonst.github.io/push-workshop/)
```
Comment (macdonst) [inline README.md]: There are actually two tutorials. The US one uses node-gcm/node-apns to send push data. The EU one uses the PhoneGap CLI http://macdonst.github.io/push-workshop-eu/, so I'd put in links to both.

Comment (fredgalvao) [inline README.md]: Interesting. I saw both links you posted, and I have them kept "at home" too, for a while now. But upon simple inspection, both use the same repo as a base, and thus I thought they were the same.

If they're both usefull each on it's own, I'll add both then with the info on where they differ.

Comment (macdonst) [inline README.md]: Yeah, they are mostly the same but the EU one is better for getting started as no need to setup certs/provs etc.

**Comment by macdonst** [comment]
Thanks @fredgalvao

**Pull request merged**